<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CurrentSessionSettingsDialog</class>
 <widget class="QDialog" name="CurrentSessionSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Current session settings dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QListWidget" name="listSettings">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>180</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="viewMode">
        <enum>QListView::ListMode</enum>
       </property>
       <item>
        <property name="text">
         <string>Subtitles</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Audio effects</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QListWidget" name="listSubTrack">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>120</width>
               <height>16777215</height>
              </size>
             </property>
             <item>
              <property name="text">
               <string>1st track</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2nd track</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>3rd track</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QStackedWidget" name="stackedWidget2">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="stackedWidgetPage1">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QFormLayout" name="formLayout">
               <item row="0" column="0">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Subtitles file path</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QLineEdit" name="lineEditSubFilePath1"/>
                 </item>
                 <item>
                  <widget class="QPushButton" name="buttonBrowseSub1">
                   <property name="maximumSize">
                    <size>
                     <width>24</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>...</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Enabled</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QCheckBox" name="checkBoxSubEnabled1">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QFormLayout" name="formLayout_2">
               <item row="0" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Subtitles appear after audio (ms)</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QDoubleSpinBox" name="spinBoxSubShift1">
                   <property name="minimumSize">
                    <size>
                     <width>80</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="minimum">
                    <double>-2000.000000000000000</double>
                   </property>
                   <property name="maximum">
                    <double>2000.000000000000000</double>
                   </property>
                   <property name="singleStep">
                    <double>0.100000000000000</double>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>274</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_4">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <layout class="QFormLayout" name="formLayout_3">
               <item row="0" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Subtitles file path</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <item>
                  <widget class="QLineEdit" name="lineEditSubFilePath2"/>
                 </item>
                 <item>
                  <widget class="QPushButton" name="buttonBrowseSub1_2">
                   <property name="maximumSize">
                    <size>
                     <width>24</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>...</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Enabled</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QCheckBox" name="checkBoxSubEnabled2">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QFormLayout" name="formLayout_4">
               <item row="0" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Subtitles appear after audio (ms)</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <layout class="QHBoxLayout" name="horizontalLayout_7">
                 <item>
                  <widget class="QDoubleSpinBox" name="spinBoxSubShift2">
                   <property name="minimumSize">
                    <size>
                     <width>80</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="minimum">
                    <double>-2000.000000000000000</double>
                   </property>
                   <property name="maximum">
                    <double>2000.000000000000000</double>
                   </property>
                   <property name="singleStep">
                    <double>0.100000000000000</double>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_3">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>274</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_5">
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <layout class="QFormLayout" name="formLayout_5">
               <item row="0" column="0">
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>Subtitles file path</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <layout class="QHBoxLayout" name="horizontalLayout_8">
                 <item>
                  <widget class="QLineEdit" name="lineEditSubFilePath3"/>
                 </item>
                 <item>
                  <widget class="QPushButton" name="buttonBrowseSub1_3">
                   <property name="maximumSize">
                    <size>
                     <width>24</width>
                     <height>16777215</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>...</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>Enabled</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QCheckBox" name="checkBoxSubEnabled3">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QFormLayout" name="formLayout_6">
               <item row="0" column="0">
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>Subtitles appear after audio (ms)</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <layout class="QHBoxLayout" name="horizontalLayout_9">
                 <item>
                  <widget class="QDoubleSpinBox" name="spinBoxSubShift3">
                   <property name="minimumSize">
                    <size>
                     <width>80</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="minimum">
                    <double>-2000.000000000000000</double>
                   </property>
                   <property name="maximum">
                    <double>2000.000000000000000</double>
                   </property>
                   <property name="singleStep">
                    <double>0.100000000000000</double>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_4">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer_4">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>274</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <layout class="QFormLayout" name="formLayout_7">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Volume (%)</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="spinBoxVolume">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximum">
              <number>200</number>
             </property>
             <property name="singleStep">
              <number>10</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>335</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_3"/>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>387</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSave">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>listSettings</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listSubTrack</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>stackedWidget2</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>436</x>
     <y>147</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSave</sender>
   <signal>clicked()</signal>
   <receiver>CurrentSessionSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>729</x>
     <y>447</y>
    </hint>
    <hint type="destinationlabel">
     <x>799</x>
     <y>445</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>CurrentSessionSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>643</x>
     <y>441</y>
    </hint>
    <hint type="destinationlabel">
     <x>655</x>
     <y>483</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
